app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] According to the requirements, it is imperative to include the `:confirmable` feature for users. However, it appears that this feature is currently missing. Please add this feature and ensure its proper functionality. The ability to confirm email during the sign-up process should be available when signing in.